src/python/grpcio_tests/tests/interop/methods.py

```diff
@@ -291,36 +315,89 @@ def _empty_stream(stub):
 
 
 def _status_code_and_message(stub):
-  message = 'test status message'
-  code = 2
-  status = grpc.StatusCode.UNKNOWN # code = 2
+  # Function wide constants
+  _MESSAGE = 'test status message'
+  _CODE = 2
+  _STATUS = grpc.StatusCode.UNKNOWN # code = 2
+
+  def ValidateStatusCodeAndMessage(response):
```

Comment:
I think it's worth pulling this behavior out to module-scope even if it means adding another parameter or two.